src/ray/test/run_gcs_tests.sh

```diff
@@ -14,15 +16,15 @@ if [[ "${RAY_USE_NEW_GCS}" = "on" ]]; then
         --loadmodule ./src/ray/gcs/redis_module/libray_redis_module.so \
         --port 6379 &
 else
-    ./src/ray/thirdparty/redis/src/redis-server \
+    ./bazel-genfiles/ray_pkg/ray/core/src/ray/thirdparty/redis/src/redis-server \
         --loglevel warning \
-        --loadmodule ./src/ray/gcs/redis_module/libray_redis_module.so \
+        --loadmodule ./bazel-bin/ray_redis_module.so \
         --port 6379 &
 fi
 sleep 1s
 
-./src/ray/gcs/client_test
-./src/ray/gcs/asio_test
+./bazel-bin/gcs_client_test
```

Comment:
The plan is to shift this to "bazel test" in the next PR, so all these testing scripts can go away.